package.json

```diff
@@ -120,5 +107,29 @@
       "prettier-eslint --print-width 120 --single-quote --tab-width 4 --write",
       "git add"
     ]
+  },
+  "jest": {
```

Comment:
Is it possible to break this out in a separate config file like we did for karma?

Reply:
I think this actually needs to be in package.json